wasmtime-jni/src/io_github_kawamuray_wasmtime_Global/imp.rs

```diff
@@ -0,0 +1,69 @@
+use super::JniGlobal;
+use crate::errors::{self, Result};
+use crate::store::StoreData;
+use crate::{interop, wval};
+use jni::objects::{JObject};
+use jni::sys::{jlong, jobject};
+use jni::{JNIEnv};
+use wasmtime::{Global, Store};
+
+pub(super) struct JniGlobalImpl;
+
+impl<'a> JniGlobal<'a> for JniGlobalImpl {
+    type Error = errors::Error;
+
+    fn native_get(
+        env: &JNIEnv,
+        this: JObject,
+        store_ptr: jlong,
+    ) -> Result<jobject, Self::Error> {
+        let mut store = interop::ref_from_raw::<Store<StoreData>>(store_ptr)?;
+        let global = interop::get_inner::<Global>(&env, this)?;
+
+        let val = global.get(&mut *store);
+
+        match wval::into_java(env, val.clone()) {
```

Comment:
This can be simplified by using `?` operator like this:

`Ok(wval::into_java(env, val)?.into_inner())`

Also, I don't think we need to create a clone of val here?